src/components/alarm/AlarmController.h

```diff
@@ -29,42 +29,57 @@ namespace Pinetime {
   namespace Controllers {
     class AlarmController {
     public:
-      AlarmController(Controllers::DateTime& dateTimeController);
+      AlarmController(Controllers::DateTime& dateTimeController, Controllers::FS& fs);
 
       void Init(System::SystemTask* systemTask);
+      void SaveAlarm();
       void SetAlarmTime(uint8_t alarmHr, uint8_t alarmMin);
       void ScheduleAlarm();
       void DisableAlarm();
       void SetOffAlarmNow();
       uint32_t SecondsToAlarm() const;
       void StopAlerting();
-      enum class AlarmState { Not_Set, Set, Alerting };
```

Comment:
Why was this `enum` replaced by 2 bools?

Reply:
I think to have the split between `bool isEnabled` inside the stored settings and the temporary `bool isAlarming`, which shouldn't be stored

Reply:
Fair enough, thanks!